src/pages/dmt5/_trade-control.js

```diff
@@ -0,0 +1,182 @@
+import { graphql, useStaticQuery } from 'gatsby'
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Flex, SectionContainer } from 'components/containers'
```

Comment:
shouldnt we hide this section, since swap calculator is not available yet.

Reply:
Designers asked me to hode the buttons only. If I wanna hide the whole section it requires new design I think,